### DISCUSSION TITLE
tsconfig 按照文档配置后报错，而且也没法按需引入，环境哪里出问题了吗

### DISCUSSION THREAD

**Discussion by klren0312** [zouyaoji/vue-cesium#629] [Q&A]
- vite: 5.4.10
- typescript: 5.6.2
- vue: 3.5.12

![image](https://github.com/user-attachments/assets/d4f603eb-39ad-40d2-b58f-e6db88b1778f)
![image](https://github.com/user-attachments/assets/07f308d0-0cf0-401d-8ba0-b2fc7c21dace)

**Comment by zouyaoji** [comment]
types的路径也写全试试

**Reply by klren0312** [reply]
写全了 也不行哎 我先换成原生顶上去了 哈哈 准备用这个快速开发的

**Reply by zouyaoji** [reply]
表示没遇到过哈。空了可以发个精简的重现问题的工程来哈，我抽时间看看。

**Comment by 4uansi** [comment]
这个问题解决了吗，遇到了同样的问题

**Reply by klren0312** [reply]
改用原生的了 没时间看。。。
